Application/EdFi.Ods.Common/Infrastructure/Extensibility/EntityExtensionRegistrar.cs

```diff
@@ -53,7 +53,9 @@ public EntityExtensionRegistrar(IEnumerable<Assembly> extensionAssemblies, IDoma
         /// <param name="aggregateExtensionEntity">The <see cref="EdFi.Ods.Common.Models.Domain.Entity" /> representing the Ed-Fi extension entity added to an Ed-Fi standard aggregate.</param>
         private void RegisterAggregateExtensionEntity(Type edFiStandardEntityType, Entity aggregateExtensionEntity)
         {
-            _logger.Debug($"updating {nameof(edFiStandardEntityType)} with aggregate extension {aggregateExtensionEntity.FullName}");
+            _logger.Debug(
+                $"updating {nameof(edFiStandardEntityType)} with aggregate extension {aggregateExtensionEntity.FullName}");
```

Comment:
Add single quotes around the parameters in the log message.

Reply:
done